src/IRac.cpp

```diff
@@ -1539,6 +1539,7 @@ void IRac::mirage(IRMirageAc *ac,
 /// @note Clock can only be set in 10 minute increments. i.e. % 10.
 void IRac::mitsubishi(IRMitsubishiAC *ac,
                       const bool on, const stdAc::opmode_t mode,
+                      const bool iSave10C,
```

Comment:
Rename to "econo", and move it to after the "quiet" parameter.
You also need the special comment line above the function for doxygen to document it.
e.g.
```c++
/// @param[in] econo Toggle the device's economical mode.
```